src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
```

Comment:
Ensure that the elements '.spider' and '.wall' exist in the DOM before this script runs. If they are not present or not fully loaded, `document.querySelector` will return `null`, leading to errors in subsequent lines.